### PR TITLE
Update redis to 0.15 and r2d2_redis to 0.13

### PR DESCRIPTION
**Pull request by rnestler**
## Checklist

- [na] Tests added if applicable
- [na] README updated if applicable
- [x] CHANGELOG updated if applicable
- [x] If a commit includes a breaking change, include the string `[breaking-change]` somewhere in the commit message

### CODE REVIEW

**Comment by rnestler** [comment]
Is a semver breaking update of redis nad r2d2_redis breaking? Do we expose it anywhere?

**Comment by rnestler** [comment]
Ah well it seems to need a bump in our MSRV :slightly_smiling_face:

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Possible log types:
 
 ### Unreleased
 
+- [changed] Update redis to 0.15 and r2d2\_redis to 0.13 (#105)
```
Comment (dbrgn) [inline CHANGELOG.md:16]: Do you want to mention the MSRV change as well?

Comment (rnestler) [reply CHANGELOG.md:16]: Well we don't have a MSRV according to our README, that's why I didn't mention it.

Comment (dbrgn) [reply CHANGELOG.md:16]: Ah, true.

**Pull request merged**